ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -27600,16 +27600,6 @@ periodics:
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-aws-ipi-disconnected-private-p3-f28-long-duration-cloud
-  reporter_config:
```

Comment:
Unexpected removal of existing configs.

Reply:
Sorry, I thought it was removed by someone else, I don't have sync here.